MainDlg.cpp

```diff
@@ -390,3 +390,67 @@ LRESULT CMainDlg::OnEnumSymbolsDone(UINT uMsg, WPARAM wParam, LPARAM lParam) {
 
     return 0;
 }
+const COMDLG_FILTERSPEC file_types[] = {
+    {L"Supported Files (*.dll;*.exe)", L"*.dll;*.exe"},
+    {L"All Files (*.*)", L"*.*"},
+};
+
+LRESULT CMainDlg::OnBnClickedPickfile(WORD /*wNotifyCode*/,
+                                    WORD /*wID*/,
+                                    HWND /*hWndCtl*/,
+                                    BOOL& /*bHandled*/) {
+    // https://learn.microsoft.com/en-us/windows/win32/shell/common-file-dialog#basic-usage
+
+    IFileDialog* pfd = NULL;
```

Comment:
Use smart pointers (CComPtr) to avoid memory leaks.

Reply:
but it is released at the end

Reply:
ohh nvm